tests/models_tests/models_test.go

```diff
@@ -505,3 +505,24 @@ func TestGetCohortDefinitionById(t *testing.T) {
 		t.Errorf("Expected data not found")
 	}
 }
+
+func TestRetrieveDataByOriginalCohortAndNewCohort(t *testing.T) {
+	setUp(t)
+	originalCohortSize := secondLargestCohort.Id
```

Comment:
is this a copy/paste error? Assigning "Id" to a size variable looks strange here

Reply:
yup good catch, should have been `secondLargestCohort.cohortSize`